scripts/emulated-performance.ps1

```diff
@@ -276,6 +282,17 @@ if ($LogProfile -ne "None") {
 $Platform = $IsWindows ? "windows" : "linux"
 $PlatformName = (($IsWindows ? "Windows" : "Linux") + "_$($Arch)_$($Tls)")
 
+if ($BaseRandomSeed -eq "") {
+    for ($i = 0; $i -lt 3; $i++) {
+        $BaseRandomSeed += $(Get-Random).ToString('x8')
+    }
+    $BaseRandomSeed += $(Get-Random).ToString('x8').Substring(0,6)
+    # This gives 15 bytes of random seed, and the last byte will be
+    # the iteration count.
+}
+
+Write-Debug "BaseRandomSeed: $($BaseRandomSeed)"
```

Comment:
This likely need to be a `Write-Host` because we always want to see it.

Reply:
Agreed